packages/cart/src/models/adjustment-line.ts

```diff
@@ -0,0 +1,41 @@
+import { DAL } from "@medusajs/types"
+import { OptionalProps, PrimaryKey, Property } from "@mikro-orm/core"
+
+type OptionalAdjustmentLineProps = DAL.EntityDateColumns // TODO: To be revisited when more clear
+
+export default class AdjustmentLine {
```

Comment:
suggestion: Add a comment that this is not meant to be a db entity (for future reference)

Reply:
ref - https://mikro-orm.io/docs/inheritance-mapping